src/controllers/authController.ts

```diff
@@ -0,0 +1,49 @@
+import { Request, Response } from 'express';
+import { User } from '../entities/User';;
+import bcrypt from 'bcrypt';
+import { getRepository } from 'typeorm';
+
+
+class UserController {
+    static registerUser = async (req: Request, res: Response) => {
+        const { firstName, lastName, email, password, gender, phoneNumber, userType, status, verified, photoUrl } = req.body;
```

Comment:
Can you consider giving the verified an initial value of false in the user entity so that their cannot be a need of inputting it, you can also make it to status and give it an initial value pf active so that when creating a user, a user can be having verification false and status active without a need of putting that in the request.